File: Modul-2/Soal-5/PRAK205-2410817220023-CLARISSADHEAALLISYA.c
#include <stdio.h>
#include <math.h>

int main(){
    // test case 1
    float A, B, C, keliling, luas;
    scanf("%f", &A); scanf("%f", &B);

    C = sqrt((B * B) - (A * A));
    keliling = A + B + C;
    luas = 0.5 * C * A;

    printf("Alas = %.f cm\n", C);
    printf("Tinggi = %.f cm\n", A);
    printf("Keliling = %.f cm \n", keliling);
    printf("Luas = %.f cm^2\n", luas);
    printf("\n");

    // test case 2
    float A2, B2, C2, keliling2, luas2;
    scanf("%f", &A2); scanf("%f", &B2);

    C2 = sqrt((B2 * B2) - (A2 * A2));
    keliling2 = A2 + B2 + C2;
    luas2 = 0.5 * C2 * A2;

    printf("Alas = %.f cm\n", C2);
    printf("Tinggi = %.f cm\n", A2);
    printf("Keliling = %.f cm \n", keliling2);
    printf("Luas = %.f cm^2\n", luas2);
    return 0;
}

File: Modul-2/Soal-5/PRAK205-2410817220023-CLARISSADHEAALLISYA.py
import math
# test case 1
A, B = map(float, input().split())
C = math.sqrt((B * B) - (A * A))
keliling = A + B + C
luas = 0.5 * C * A

print("Alas = %.0f cm "%(C))
print("Tinggi = %.0f cm "%(A))
print("Keliling = %.0f cm "%(keliling))
print("Luas = %.0f cm^2 "%(luas))
print("")

# test case 2
A2 = float(input())
B2 = float(input())
C2 = math.sqrt((B2 * B2) - (A2 * A2))
keliling2 = A2 + B2 + C2
luas2 = 0.5 * C2 * A2

print("Alas = %.0f cm "%(C2))
print("Tinggi = %.0f cm "%(A2))
print("Keliling = %.0f cm "%(keliling2))
print("Luas = %.0f cm^2 "%(luas2))